wearereasonablepeople/warped-components
# Add type declarations

# Motivation
With type declarations we can get a lot of useful information mostly in the inferred `props` for warped components.

![image](https://user-images.githubusercontent.com/1294893/44657864-bba8f680-a9fe-11e8-9f3f-be8f8f688a28.png)

See https://github.com/paolobueno/warped-ts-example for the test/example.

# Changes
- Add typings
- Include typings in published package

# Verification instructions
- This branch is available at `npm install paolobueno/gitpkg-registry#warped-reducers-v1.0.5-gitpkg` for testing
- See https://github.com/paolobueno/warped-ts-example for a working TS example
- Also try using this version on top of a previous one and see extra information supplied by vscode's language services
